torch/csrc/autograd/FunctionsManual.cpp

```diff
@@ -2270,6 +2270,66 @@ Tensor eig_backward(const std::vector<torch::autograd::Variable> &grads, const T
   return at::linalg_solve(Uh, at::matmul(U_contrib, Uh) + D_contrib * Uh);
 }
 
+Tensor linalg_eig_backward(const std::vector<torch::autograd::Variable> &grads,
+                           const Tensor& self,
+                           const Tensor& lambda,
+                           const Tensor& v) {
```

Comment:
We need a comment that lambda represents eigenvalues and v represents eigenvectors. Or better use `const Tensor& eigenvalues, const Tensor& eigenvectors` and then alias in code to a shorter name?

Reply:
What about using L and V, as we do in the documentation?

I used this notation in the mathematical explanation  below as I think it's a rather natural one. In that comment it's clear that V is the matrix and L the eigenvalues.

Reply:
Using L and V as in documentation sounds good.